com.unity.perception/Runtime/GroundTruth/Labeling/LabelEntryMatchCache.cs

```diff
@@ -55,10 +73,19 @@ void IGroundTruthGenerator.SetupMaterialProperties(MaterialPropertyBlock mpb, Re
             }
         }
 
+        /// <inheritdoc/>
```

Comment:
Just a note: the package validation suite is complaining about the lack of an XML doc on IGroundTruthGenerator.SetupMaterialProperties in this class, even though technically the implementation of this interface is private (which is a thing I didn't even know you could do)